xtypes.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
+    cpp-peglib:
+        type: git
+        url: https://github.com/jparisu/cpp-peglib.git
```

Comment:
Why is this not pointing to the main cpp-peglib repo?

Reply:
Because is this version the one that was being used as submodule, and I suppose the one it wants to be used.

Reply:
I mean using as url https://github.com/yhirose/cpp-peglib

Reply:
My fault. :+1: